frontend/src/components/dialogs/BlueprintSelectorDialog.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+    <ff-dialog ref="dialog" data-el="blueprint-selector-dialog" class="blueprints-selector-dialog" header="Choose a Blueprint">
+        <template #default>
+            <section class="blueprints-container w-full md:w-full lg:w-2/5 xl:w-2/5 2xl:w-2/5">
+                <div class="header hidden 2xl:block xl:block lg:block">
+                    <p>To get started, we have a collection of pre-built flow templates that you can use as a starting point for your Node-RED Instance.</p>
+                </div>
+                <div class="blueprint-selection-wrapper">
+                    <BlueprintSelection
+                        :blueprints="blueprints"
+                        :with-header="false"
+                        :preview-tiles="false"
+                        :active-blueprint="currentBlueprint"
+                        @selected="onBlueprintSelected"
+                    />
+                </div>
+            </section>
+            <section class="flow-viewer-container hidden 2xl:block xl:block lg:block w-full md:w-full lg:w-3/5 xl:w-3/5 2xl:w-3/5">
+                <div ref="viewer" class="viewer" @click.stop.prevent />
+            </section>
+        </template>
+        <template #actions>
+            <div class="flex justify-end">
+                <ff-button kind="secondary" data-action="dialog-cancel" @click="closeDialog">Cancel</ff-button>
+                <ff-button data-action="dialog-confirm" @click="confirmSelection">Confirm Changes</ff-button>
```

Comment:
```suggestion
                <ff-button data-action="dialog-confirm" @click="confirmSelection">Confirm Choice</ff-button>
```

Suggestion: "Confirm Choice" (since nothing was _really_ changed)